src/purchase_listinator/endpoints/http/purchase_list.clj

```diff
@@ -21,7 +22,8 @@
             (let [user-id (adapters.misc/string->uuid user-id)]
               (flows.purchase-list/get-lists user-id datomic)))
           misc.http/->Error
-          misc.http/->Success))
+          #(-> (map adapters.out.purchase-list/internal->wire %)
```

Comment:
Applying the adapter so the function respects the expected schema output.

Reply:
Perfect!
You can use the thread first and thread last to let it a little cleaner, also, if you want, you can change the branch for the `misc.http/default-branch` macro:

```clojure
(misc.http/default-branch
    (misc.either/try-right
      (-> (adapters.misc/string->uuid user-id)
          (flows.purchase-list/get-lists datomic)
          (->> (map adapters.out.purchase-list/internal->wire))))))
```